data/hazards.txt

```diff
@@ -124,3 +124,20 @@ hazard "Rulei Gravity Field"
 
 effect "gravity well"
 	"sound" "gravity well"
+
+
+hazard "Core Sheltered Heat"
+	"duration" 1
+	"strength" 0.65 0.85
```

Comment:
`(rounded slightly)`

Reply:
You have entirely missed the point - I'm not referring to the *current, exact* balance, I am referring to the math setup for the *process* of doing the balancing.
i.e. If you change thing A, for Situation B, how much do you need to change other thing C to preserve situation D?

And since there is more balancing to be done, the *process* is still in use, and so this scaling will not be changing at this time.

Reply:
Sorry, let me just paste a spreadsheet in here... /sarcasm

This whole argument is irrelevant at this point in time. It will be revisited when the balancing process is complete. End of.